config.js

```diff
@@ -9,6 +9,8 @@ const localEnvPath = path.resolve(process.cwd(), '.env.local')
 const envPath = path.resolve(process.cwd(), `.env.${env}`)
 const defaultEnvPath = path.resolve(process.cwd(), '.env')
 
+console.log('localEnvPath', localEnvPath, 'envPath', envPath, 'defaultEnvPath', defaultEnvPath)
```

Comment:
Let's remove console logs (we already have quite a few unwanted debug logs statements in our production build)